### PR TITLE
Completed data-structures-stacks.

### PR DESCRIPTION
**Pull request by stefancarrera**
data-stacks

### CODE REVIEW

File: data-structures-stacks/count-values.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* exported countValues */
+
+function countValues(stack) {
+  let count = 0;
+  if (stack.peek() !== 'Stack { <empty> }') {
```
Comment (scbowler) [inline data-structures-stacks/count-values.js:5]: `.peek` will never be `'Stack { <empty> }'` but it can be `undefined`

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking great, just one update needed

**Comment by stefancarrera** [comment]
data-stacks2

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback! Looks good! 👍

**Pull request merged**